gluon/scheduler.py

```diff
@@ -978,14 +979,17 @@ def wrapped_pop_task(self):
         contention and retries `pop_task` after 0.5 seconds
         """
         db = self.db
-        db.commit()  # for MySQL only; FIXME: Niphlod, still needed? could avoid when not MySQL?
```

Comment:
doesn't cost much, so leave it, also, it implicitely checks if the connection is usable ;-)

Reply:
ok but I'll have to surround it in a try catch. The reason I removed it was because it could fail if the connection was down and I didn't see why it was needed.

Reply:
it is already in a wrapped function...if it fails let it fail